packages/expo-modules-autolinking/src/platforms/android.ts

```diff
@@ -4,6 +4,9 @@ import path from 'path';
 
 import { ModuleDescriptorAndroid, PackageRevision } from '../types';
 
+// Only modules in this whitelist could apply gradle plugins automatically
+const GRADLE_PLUGIN_WHITELISTS = ['expo-dev-launcher'];
```

Comment:
I'm not sure if we should whitelist which plugins should be automatically applied. If the module contains a plugin section in its config, it probably requires it to run correctly. Therefore, we should apply it.

Reply:
since it's a new feature, i'm hesitant to enable this for all modules, though it should be a good feature. or maybe we enable for all modules but adding more conspicuous logs than `println "Applying gradle plugin '${Colors.GREEN}${modulePlugin.id}${Colors.RESET}' (${module.name}@${module.version})"`, somehow adding a emoji.

does that sound good to you?

Reply:
I like that idea. Let's add logs for now ;)